builder/dockerfile/instructions/parse.go

```diff
@@ -273,6 +274,14 @@ func parseFrom(req parseRequest) (*Stage, error) {
 	}
 
 	flPlatform := req.flags.AddString("platform", "")
+	SpecPlatform := &specs.Platform{}
+	if flPlatform.Value != "" {
```

Comment:
Shouldn't this be after `req.flags.Parse()`?

Reply:
Right. Will push an update soon...